app/[lang]/layout.tsx

```diff
@@ -0,0 +1,36 @@
+import "./globals.css";
+
+import type { Metadata } from "next";
+import { Geist, Geist_Mono } from "next/font/google";
+import { ReactNode } from "react";
+
+const geistSans = Geist({
+  variable: "--font-geist-sans",
+  subsets: ["latin"],
+});
+
+const geistMono = Geist_Mono({
+  variable: "--font-geist-mono",
+  subsets: ["latin"],
+});
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: ReactNode;
+}>) {
```

Comment:
```suggestion
}: PropsWithChildren<{}>) {
```

Reply:
加插件规则？

```
// ... existing code ...
      "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
      "no-restricted-syntax": [
        "error",
        {
          "selector": "TSPropertySignature[key.name='children']",
          "message": "Please use PropsWithChildren<T> instead of defining children manually"
        }
      ],
// ... existing code ...
```

Reply:
> 加插件规则？
> 
> ```
> // ... existing code ...
>       "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
>       "no-restricted-syntax": [
>         "error",
>         {
>           "selector": "TSPropertySignature[key.name='children']",
>           "message": "Please use PropsWithChildren<T> instead of defining children manually"
>         }
>       ],
> // ... existing code ...
> ```

还可以这样？加